public/js/gulliver.es6.js

```diff
@@ -5,6 +5,7 @@
 
 /* eslint-env browser */
 
+import '../../node_modules/whatwg-fetch/fetch.js';
```

Comment:
Yeah, it looks a bit funny because our build process is weird (the `*.es6.js` files don't need to be in the `public` directory): `rollup` merges all the modules together into `gulliver.js`, which is the only file that's needed in production. (Well, `lighthouse-chart.js` is needed too, but we'll move that eventually.)